<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDlg</class>
 <widget class="QDialog" name="OptionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtNote Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../main.qrc">
    <normaloff>:/icons/options</normaloff>:/icons/options</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Storage Priority</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="priorityView">
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="movement">
         <enum>QListView::Snap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckAskDel">
     <property name="text">
      <string>Delete notes with confirmation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckAutostart">
     <property name="text">
      <string>Start with system</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblNoteFromSel">
       <property name="text">
        <string>Note from selection shortcut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ShortcutEdit" name="leNoteFromSelShortcut"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Amount of notes to show in the tray menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spMenuNotesAmount"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutEdit</class>
   <extends>QLineEdit</extends>
   <header>src/shortcutedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>priorityView</tabstop>
  <tabstop>ckAskDel</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
